### DISCUSSION TITLE
User Management

### DISCUSSION THREAD

**Discussion by x-du** [FlowiseAI/Flowise#2962] [General]
Currently, On Flowise, you can only have one admin account. It would be nice to have user management on Flowise, so that we can have multiple users, and each user only sees their own content (flows and agentflows). 

If you know how to make it work,, please let me know.

**Comment by severfire** [comment]
i think it would be great idea to have it, also giving different access rights via ACL

**Comment by MAmmarShahid** [comment]
Great idea. Should be implemented.
How can we achieve this using current version?
